### DISCUSSION TITLE
How to freeze a multi-task DPA model

### DISCUSSION THREAD

**Discussion by EricGaoOm** [deepmodeling/deepmd-kit#4393] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/173cb3eb-c09d-41f5-911b-56247980eb08)
After training the multi-task model, I try to use  "dp --pt freeze" to freeze the checkpoint, however, the script told me to add a head. I also tried  "dp --pt freeze -o frozen.pth", it just not work.

**Comment by njzjz** [comment, marked as answer]
Please follow the documentation: https://docs.deepmodeling.com/projects/deepmd/en/latest/freeze/freeze.html

**Reply by EricGaoOm** [reply]
Many thanks，it works well
